inference-engine/src/low_precision_transformations/src/common/reshape.cpp

```diff
@@ -28,65 +28,109 @@ void ReshapeTransformation::registerMatcherIn(GraphRewrite &pass, Transformation
 void reshapeDequantizationConstant(const std::shared_ptr<opset1::Reshape>& reshape) {
     const FakeQuantizeDequantization dequantization = NetworkHelper::getDequantization(reshape, 0);
     if (dequantization.multiply->get_input_node_ptr(1)->get_output_shape(0).size() > 1ul) {
+        // Reshape Subtract or Multiply operation Constant.
```

Comment:
overcomplicated solution. please look into the same problem I resolved here https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/src/transformations/common_optimizations/fq_reshape_fusion.cpp

Reply:
behavior is valid. logic that i suggested is incomplete for the use cases of @eshoguli code. my bad, sorry.